### DISCUSSION TITLE
restart chgnet training (finetune)

### DISCUSSION THREAD

**Discussion by chtchelkatchev** [CederGroupHub/chgnet#172] [General]
Is there possibility to restart training (finetune) of chgnet? 
Due to supercomputer limitations I can use A100 gpus only 24 hours, after that I must restart any calculations. 24 hours is not enough sometimes for training (finetune) chgnet. 

Is there possiblity to use several gpu for training (if this speeds up)?

**Comment by chtchelkatchev** [comment]
Four finetune I tried the following procedure (is it correct?):
trainer = Trainer(
    model=chgnet,
    targets="efsm",
    optimizer="Adam",
    criterion="MSE",
    learning_rate=8e-3,
    epochs=1200,
    use_device="cuda",
    starting_epoch=803,
    scheduler="CosRestartLR",
)

Here 803--is the last epoch of the previous finetuning. CosRestartLR repalces CosLR.

**Comment by bowen-bd** [comment]
You can try directly [loading the trainer](https://github.com/CederGroupHub/chgnet/blob/22a0dbb31271df568cf6b21d2a968ad86be8b837/chgnet/trainer/trainer.py#L648).

This way you don't have to redefine parameters.
Additionally, CosRestartLR is not related to reloading CosLR.
